README.md

```diff
@@ -305,6 +306,30 @@ method takes one argument:
 
 2. `chain` must return a value of the same Chain
 
+### ChainRec
+
+A value that implements the ChainRec specification must also implement the Chain specification.
+
+1. `T.chainRec((done, next, value) => g(done(value)), i) `is equivalent to `g(i)`
```

Comment:
Ah, right. It makes sense if we use `of` as `g`. Sorry.

Still curious if we could add more laws. Are there any laws for it in PureScript? If not, should we made up our own? Maybe we should add dependency on Monad after all, so we could use `of` and `map` in laws?

I just worry what is the value of having a method in the spec if we don't have laws that strictly restrict what it can do?